livelossplot/plot_losses.py

```diff
@@ -6,8 +6,8 @@
 
 
 class PlotLosses:
-    def __init__(self, outputs: List[Type[BaseOutput]] = [Matplotlib()]):
-        self.logger = MainLogger()
+    def __init__(self, outputs: List[Type[BaseOutput]] = (Matplotlib(),), **kwargs):
```

Comment:
I guess this list to a tuple is to avoid copying (the error you send a message). However, I find it not a good practice to use tuples for such things of varying length (it should work well with append).

Reply:
We don't want to modify it during runtime? If we want, it should be list. In other case error with mutable default is a worse practice than tuple for things with varying length.